spec/requests/queries/user_recipes/bought_recipes_by_user_id_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+module Queries
+  RSpec.describe Types::QueryType, type: :request do
+    before :each do
+      @user = create(:user, :with_recipes)
+      @recipe_one = @user.recipes.first
+      @recipe_two = @user.recipes.last
+      @buyer = create(:user)
+      @user_recipe_one = UserRecipe.create(user_id: @buyer.id, recipe_id: @recipe_one.id, amount_donated: 5.50)
```

Comment:
Ooh that's a good sad path "A user cannot buy their own Recipe"
I'll probably make a new or for that one since it has to do more with createUserRecipe